applets/wncklet/org.mate.panel.applet.window-list.gschema.xml.in

```diff
@@ -20,5 +20,10 @@
       <summary>Move windows to current workspace when unminimized</summary>
       <description>If true, then when unminimizing a window, move it to the current workspace. Otherwise, switch to the workspace of the window.</description>
     </key>
+    <key name="scroll-enabled" type="b">
+      <default>true</default>
+      <summary>Whether to enable mouse scrolling in the switch window list</summary>
+      <description>If true, then when unminimizing a window, move it to the current workspace. Otherwise, switch to the workspace of the window.</description>
```

Comment:
The description should be updated.

Reply:
Thank you for reminding me. This is my negligence. The description has been updated